Error: 'Config' object has no attribute 'get_movie_show_or_collection'

I am getting the error when using `tmdb_background`. The TMDb ID is correct, as it works for other fields that use it. It happens for all collections that have this attribute. 
```
[2021-02-11 07:50:42,840] [util.py:575]               [DEBUG]    | Traceback (most recent call last):                                                                 |
                                                                 |   File "/home9/calboi/Plex-Meta-Manager/modules/config.py", line 564, in update_libraries
                                                                 |     elif method_name == "tmdb_background":                              backgrounds_found.append(("url", "{}{}".format(self.TMDb.image_url, self.get_movie_show_or_collection(util.regex_first_int(collections[c][m], "TMDb ID"), library.is_movie).poster_path), method_name))
                                                                 | AttributeError: 'Config' object has no attribute 'get_movie_show_or_collection'
                                                                 | 
[2021-02-11 07:50:42,840] [config.py:1029]            [ERROR]    | Unknown Error: 'Config' object has no attribute 'get_movie_show_or_collection'
```

Also, is it possible to add collection details from a movie or show? If I have a collection built with `tmdb_collection` using a collection ID, it gives me the following error when I set the `tmdb_poster` using a show ID: `No Movie or Collection found for TMDb ID ###`.

--- a/README.md
+++ b/README.md
@@ -1,5 +1,5 @@
 # Plex Meta Manager
-#### Version 1.0.1
+#### Version 1.0.2
 
 The original concept for Plex Meta Manager is [Plex Auto Collections](https://github.com/mza921/Plex-Auto-Collections), but this is rewritten from the ground up to be able to include a scheduler, metadata edits, multiple libraries, and logging. Plex Meta Manager is a Python 3 script that can be continuously run using YAML configuration files to update on a schedule the metadata of the movies, shows, and collections in your libraries as well as automatically build collections based on various methods all detailed in the wiki. Some collection examples that the script can automatically build and update daily include Plex Based Searches like actor, genre, or studio collections or Collections based on TMDb, IMDb, Trakt, TVDb, AniDB, or MyAnimeList lists and various other services.
 

--- a/modules/config.py
+++ b/modules/config.py
@@ -574,7 +574,7 @@ class Config:
                                         if os.path.exists(collections[c][m]):                               posters_found.append(("file", os.path.abspath(collections[c][m]), method_name))
                                         else:                                                               raise Failed("Collection Error: Poster Path Does Not Exist: {}".format(os.path.abspath(collections[c][m])))
                                     elif method_name == "url_background":                               backgrounds_found.append(("url", collections[c][m], method_name))
-                                    elif method_name == "tmdb_background":                              backgrounds_found.append(("url", "{}{}".format(self.TMDb.image_url, self.get_movie_show_or_collection(util.regex_first_int(collections[c][m], "TMDb ID"), library.is_movie).poster_path), method_name))
+                                    elif method_name == "tmdb_background":                              backgrounds_found.append(("url", "{}{}".format(self.TMDb.image_url, self.TMDb.get_movie_show_or_collection(util.regex_first_int(collections[c][m], "TMDb ID"), library.is_movie).poster_path), method_name))
                                     elif method_name == "file_background":
                                         if os.path.exists(collections[c][m]):                               backgrounds_found.append(("file", os.path.abspath(collections[c][m]), method_name))
                                         else:                                                               raise Failed("Collection Error: Background Path Does Not Exist: {}".format(os.path.abspath(collections[c][m])))

--- a/plex_meta_manager.py
+++ b/plex_meta_manager.py
@@ -56,7 +56,7 @@ logger.info(util.get_centered_text("| |_) | |/ _ \ \/ / | |\/| |/ _ \ __/ _` | |
 logger.info(util.get_centered_text("|  __/| |  __/>  <  | |  | |  __/ || (_| | | |  | | (_| | | | | (_| | (_| |  __/ |   "))
 logger.info(util.get_centered_text("|_|   |_|\___/_/\_\ |_|  |_|\___|\__\__,_| |_|  |_|\__,_|_| |_|\__,_|\__, |\___|_|   "))
 logger.info(util.get_centered_text("                                                                     |___/           "))
-logger.info(util.get_centered_text("    Version: 1.0.1                                                                   "))
+logger.info(util.get_centered_text("    Version: 1.0.2                                                                   "))
 util.seperator()
 
 if args.test:

